packages/libs/eda/src/lib/workspace/AnalysisPanel.tsx

```diff
@@ -312,40 +327,59 @@ export function AnalysisPanel({
               </div>
             )}
           />
-          <WorkspaceNavigation
-            heading={<></>}
-            routeBase={routeBase}
-            items={[
-              {
-                display: 'View Study Details',
-                route: `/details`,
-                exact: false,
-              },
-              {
-                display: 'Browse and Subset',
-                route: `/variables${lastVarPath}`,
-                exact: false,
-              },
-              {
-                display: 'Visualize',
-                // check whether user is at viz
-                route: location.pathname
-                  .replace(routeBase, '')
-                  .startsWith('/visualizations')
-                  ? '/visualizations'
-                  : `/visualizations${lastVizPath}`,
-                exact: false,
-              },
-              {
-                display: 'Download',
-                route: '/download',
-              },
-              {
-                display: 'Record Notes',
-                route: '/notes',
-              },
-            ]}
-          />
+          <div className="EDAWorkspaceNavigation">
+            <div className="FilterChips">
+              <FilterChipList
+                filters={filters}
+                removeFilter={(filter) =>
```

Comment:
This isn't feedback, but I thought it'd be a fun fact. This is an unnamed function. Function names (if provided) appear in stack traces which can help you quickly identify which function caused the error. This can be helpful when a function is getting passed as a prop.